apps/storybook.namekit.io/stories/Namekit/Input.stories.tsx

```diff
@@ -112,22 +112,28 @@ export const WithSlotRight: Story = {
   },
 };
 
+export const WithSlotLeftSmall: Story = {
```

Comment:
Can you help me understand the goal of this story?

It seems we didn't add a slot here as expected?

![CleanShot 2024-09-20 at 23 31 44](https://github.com/user-attachments/assets/03cd6bd7-6786-47d9-bc45-548d3f81aa3c)

Reply:
@lightwalker-eth Thanks, fixed